Makefile

```diff
@@ -278,6 +278,8 @@ $(STATUS_CLIENT_APPIMAGE): nim_status_client $(APPIMAGE_TOOL) nim-status.desktop
 
 	# General Files
 	cp bin/nim_status_client tmp/linux/dist/usr/bin
+	mkdir -p tmp/linux/dist/usr/bin/dictionaries
```

Comment:
Since we have `mkdir -p tmp/linux/dist/usr/bin` on L275, I don't think this `mkdir` is necessary, can just do the `cp -R` below.

Reply:
got it. but i did the same as on the line 288-289

Reply:
L289 uses `cp ui/i18n/*` instead of `cp-R ui/i18n` so the destination directory needs to be created first, which is done on L288. I'm not sure if that's intentional or we could change it to use `cp -R`.